Filtered_kernel/include/CGAL/Static_filtered_predicate.h

```diff
@@ -127,23 +126,29 @@ class Static_filtered_predicate {
   result_type operator()(const A1& a1, const A2& a2, const A3& a3, const A4& a4) const
```

Comment:
Could you alternatively create a static filtered predicate overload that takes a `Origin`, there is an overload that takes `Bbox_3` for example:

https://github.com/CGAL/cgal/blob/a8e87526348d2d1faa2ea893747314853ee6868d/Filtered_kernel/include/CGAL/Static_filtered_predicate.h#L90

This would then save having to implement approx, and exact on class `Origin`. Maybe your intention was to keep the code more generic.

Reply:
@mglisse 's suggestion seems better.